lib/realtime/context_cache.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Realtime.ContextCache do
+  @moduledoc """
+    Read through cache for hot database paths.
+  """
+
+  require Logger
+
+  def apply_fun(context, {fun, arity}, args) do
+    cache = cache_name(context)
+    cache_key = {{fun, arity}, args}
```

Comment:
@chasers can't we simplify the `cache_key` to just `{fun, args}`? why do we need arity if the `args` make the function unique in the key?

Reply:
Wasn't conscious really ... __ENV__.function returns `{fn_name, arity}`. 

I guess this makes it more obvious what the cache key actually is.